tests/e2e/initialization/node.go

```diff
@@ -310,6 +310,7 @@ func (n *internalNode) initValidatorConfigs(c *internalChain, persistentPeers []
 	valConfig.StateSync.Enable = false
 	valConfig.LogLevel = "info"
 	valConfig.P2P.PersistentPeers = strings.Join(persistentPeers, ",")
+	valConfig.Consensus = tmconfig.TestConsensusConfig()
```

Comment:
Note that for this change to take effect we need to backport this to `v10.x` and rebuild the chain-init image. Then, it has to be updated here:
https://github.com/osmosis-labs/osmosis/blob/8e6e4de0bc65d521185f2aa82ff5884c107bd8c5/tests/e2e/containers/config.go#L30

If the `initialization` package is updated, the `previousVersionInit` tag should be updated as well

This is a manual step to make sure that we are able to initialize genesis and configs of the previous Osmosis version correctly.

Reply:
ahh that makes sense, I was confused why this was having no impact.

Reply:
Can you lead applying this, post this PR merge?

Reply:
I already pushed an image with the tag `v10.0.0-e2e-v3` to the Docker hub. It contains this change. Feel free to try it here

Also, I suggest not backporting this to `v10.x` via tag. We have been manually applying only `initialization` package changes so the rest of the e2e logic on `v10.x` is incompatible. As a result, backporting this via label might cause a large diff.

I opened a manual PR #2106 instead. I created the `v10.0.0-e2e-v3` image tag from that branch too